chatbot.js

```diff
@@ -0,0 +1,30 @@
+
+// I created a variable to hold my index and initialized it as an object.
+var myObject = {
+  input: ["hello","how are you?", "what is your favorite color?"],
+  output: ["Hi", "Great!", "I have so many favorites it's hard to choose one."]
+};
+
+//This is for to check that I assigned the values correctly.
+console.log(myObject);
+
+
+function reply(){
+  // I accessed object property.
+
+  var question = document.querySelector("#oranges").value.toLowerCase();
+  console.log(question);
+
+  // I wrote a conditional statement to check if the value matches with question(input) and answer(output).
+  if(myObject.input.includes(question)){
```

Comment:
This is very good. You can also only use the [`indexOf` array method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf) instead of using both `includes` and `indexOf`. `indexOf` returns -1 if the value is not found in the array so you can use this to check if the input is valid _and_ get the index of the input.

Here's an example of how to do that:
```javascript
  const index = myObject.input.indexOf(question);
  if (index !== -1) {
    let replyUser = myObject.output[index];
    document.querySelector("#output").textContent = replyUser;
  } else {
    document.querySelector("#output").textContent = "I don't understand that command. Please enter another";
  }
```

Another thing is that you should be mindful of adding whitespace when writing JavaScript to make sure your code is easy to read. Generally it helps to improve readability of your code if you follow these rules for adding whitespace: [see here](https://www.crockford.com/code.html). You should be able to format your code with your code editor so you don't have to add the spacing manually. 

For example, line 19 without spacing:
```javascript
   if(myObject.input.includes(question)){
```
With spacing added (note the space added between the `if` keyword and the `(` and the `)` and the `{`
```javascript
 if (myObject.input.includes(question)) {
```